test/mocks/targets/TargetReverter.sol

```diff
@@ -20,7 +20,7 @@ contract TargetReverter {
         revert("You shall not pass");
     }
 
-    function dueToNoPayableModifier() external pure returns (uint256) {
+    function dueToNoPayableFunction() external pure returns (uint256) {
```

Comment:
I'll rename this to `notPayable` for maximum clarity.

Reply:
Yeah, that would work, too